docs/standard/garbage-collection/large-object-heap.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "The large object heap"
+ms.custom: ""
+ms.date: "05/02/2018"
+ms.prod: ".net"
+ms.technology: dotnet-standard
+ms.topic: "article"
+helpviewer_keywords: 
+  - large object heap (LOH)"
+  - LOH
+  - "garbage collection, large object heap"
+  - "GC [.NET ], large object heap"
+author: "rpetrusha"
+ms.author: "ronpet"
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+# The large object heap
+
+The .NET Garbage Collector (GC) divides objects up into small and large objects. When an object is large, some of its attributes become more significant than if the object is small. For instance, compacting it -- that is, copying it in memory elsewhere on the heap -- can be expensive. Because of this, the .NET Garbage Collector places large objects on the large object heap (LOH). In this topic, we'll look at the large object heap in depth. We'll discuss what qualifies an object as a large object, how these large objects are collected, and what kind of performance implications large objects impose.
+
+## How an object ends up on the large object heap and how GC handles them.
+
+If an object greater than or equal to 85,000 bytes, it’s considered a large object. This number was determined by performance tuning. When an object allocation request is 85,000 or more bytes, the runtime allocates it on the large object heap.
```

Comment:
there's a grammar mistake but I am guessing someone will do an editorial pass on this so I will ignore them :)